webdriver/support/fixtures.py

```diff
@@ -142,3 +144,41 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def create_dialog(session):
+    """Create a dialog (one of "alert", "prompt", or "confirm") and provide a
+    function to validate that the dialog has been "handled" (either accepted or
+    dismissed) by returning some value."""
+
+    def create_dialog(dialog_type, text=None, result_var=None):
+        assert dialog_type in ("alert", "confirm", "prompt"), (
+               "Invalid dialog type: '%s'" % dialog_type)
+
+        if text == None:
+            text = ""
+
+        assert isinstance(text, basestring), "`text` parameter must be a string"
+
+        if result_var == None:
+            result_var = "__WEBDRIVER"
+
+        assert re.search(r"^[_$a-z$][_$a-z0-9]*$", result_var, re.IGNORECASE)
```

Comment:
It ensures that the provided string is a valid JavaScript identifier. No one
likes reading involved regular expressions, though, so I've added an assertion
message to that effect.